universal-application-tool-0.0.1/conf/routes

```diff
@@ -85,3 +85,7 @@ GET     /logout                      @org.pac4j.play.LogoutController.logout(req
 GET     /dev/seed                    controllers.dev.DatabaseSeedController.index(request: Request)
 POST    /dev/seed                    controllers.dev.DatabaseSeedController.seed()
 POST    /dev/seed/clear              controllers.dev.DatabaseSeedController.clear()
+
+# Methods for development: directly upload file to S3
+GET     /dev/fileUpload              controllers.dev.FileUploadController.index(request: Request)
+POST    /dev/fileUpload              controllers.dev.FileUploadController.upload(request: Request)
```

Comment:
Suggestion (blocking): use the conventional URL paths according to our dev guidelines

Reply:
Done.